testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py

```diff
@@ -330,3 +330,29 @@ def test_copy_prebuilt_database(params_from_base_test_setup):
 
     # Cleaning the database , tearing down
     db.deleteDB(cbl_db1)
+
+
+@pytest.mark.listener
+@pytest.mark.replication
+def test_query_arthimetic(params_from_base_test_setup):
```

Comment:
this test should be added in test_query.py, why did we add here.

Reply:
same comment here

Reply:
done